views/forum/show.ejs

```diff
@@ -213,8 +213,8 @@
                         <% } %>
                         <p class="commentOptions">
                         
-                            <a class="reply">Reply</a>
-                            <a class="like " idOfElement=<%=comment._id%> idofcomment=<%=comment._id%> idOfForum=<%=forumThread._id%> typeofelement='comment'>
+                            <a class="reply" style="cursor: pointer">Reply</a>
+                            <a class="like" style="cursor: pointer" dOfElement=<%=comment._id%> idofcomment=<%=comment._id%> idOfForum=<%=forumThread._id%> typeofelement='comment'>
```

Comment:
I think you accidentally deleted the 'i' in 'idOfElement' here.

Reply:
Oops, added it back